components/automate-ui/src/app/pages/service-groups/service-groups.component.ts

```diff
@@ -211,40 +210,34 @@ export class ServiceGroupsComponent implements OnInit, OnDestroy {
       withLatestFrom(this.route.queryParamMap),
       takeUntil(this.isDestroyed)
     )
-    .subscribe(([serviceGroups, queryParams]) => {
+    .subscribe(([serviceGroups]) => {
```

Comment:
More clear to be explicit:
```suggestion
    .subscribe(([serviceGroups, _]) => { 
```
(Otherwise someone might come along--like I almost did--and say "hey, you could just do `.subscribe(serviceGroups => {` " which at first glance seems equivalent but it is not.)